### ISSUE TITLE
Some posts aren't showing up on Mastodon

### ISSUE THREAD

**Issue by rghedin** [Automattic/wordpress-activitypub#601]
### Quick summary

I installed ActivityPub plugin in production this morning. The first published post was [perfectly posted on fediverse](https://manualdousuario.net/links-42-2023/), but the second, in which [I announced this new integration](https://manualdousuario.net/fediverso-activitypub-wordpress/) (oh, the irony 🥲), it failed.

My personal account is hosted at `mastodon.social`. Being the devs official instance, there's nothing weird there.

### Steps to reproduce

1. Publish a post on WordPress;
2. Fingers crossed that it will appear on Mastodon instances.

### What you expected to happen

All the posts should appear on fediverse (I guess?).

### What actually happened

One of two posts didn't show up on Mastodon instances.

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable

**Comment by pfefferle** [comment]
Can you try to update the fediverse post? Changing something small (whitespace) and save it.

It is really weird, the post before and after where federated but not this post...

**Comment by rghedin** [comment]
I guess I found the culprit, @pfefferle: `og:image` in `*.avif`. I just posted a third post and it showed up as expected — this time, I used a `*.png` in the link card. I made another test and noticed that Mastodon can't render `*.avif` images; when it detects a link with one of these in `og:image`, no card appears, just the raw link.

I don't know if you plan to address this use case, since WordPress still doesn't support `*.avif` by default. It would be neat, though.
